From 29cf97fe085f79ceb48943b44e34d7a0c960222e Mon Sep 17 00:00:00 2001
From: Wenbo Cao <caowenbo@mucse.com>
Date: Wed, 3 Aug 2022 12:34:54 +0800
Subject: [PATCH] net/mce: add PMD skeleton

Signed-off-by: Wenbo Cao <caowenbo@mucse.com>
---
 drivers/net/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/meson.build b/drivers/net/meson.build
index bcf488f203..2b211ece4f 100644
--- a/drivers/net/meson.build
+++ b/drivers/net/meson.build
@@ -56,6 +56,7 @@ drivers = [
         'tap',
         'thunderx',
         'txgbe',
+        'mce',
         'vdev_netvsc',
         'vhost',
         'virtio',
-- 
2.25.1

